build/stf-run-ci/tasks/main.yml

```diff
@@ -42,72 +40,13 @@
     base_dir: "{{ playbook_dir }}"
   when: base_dir | length == 0
 
-- name: Get new operator sdk
-  when: __local_build_enabled | bool or __deploy_from_bundles_enabled | bool or __deploy_loki_enabled | bool
```

Comment:
Seems like this might have actually been a bit more open than it should have. I don't think we wanted to change the golang version just because we enabled local builds or bundle deployment? Either way, I guess it's going away now :)

Reply:
Ah ok, that makes sense and might be why some of the `when` stuff got overloaded here. It might sound like we have to only soft remove some of this and leave the ability for the `__deploy_from_bundles_enabled` to get the newer operator-sdk along with the script that pulls that newer version down.

Reply:
I thought, that the loki-operator was the only thing, that uses the newer operator-sdk. Thanks for finding this.

Reply:
Once you added it, I found additional uses :)